### ISSUE TITLE
Images are not rendered if there is a variable in the alt text

### ISSUE THREAD

**Issue by szabosteve** [elastic/docs-builder#312]
### Describe the bug

If there is an attribute/variable in an image's alt text, the image is not displayed.

<img width="847" alt="Image" src="https://github.com/user-attachments/assets/fc16d48b-8a59-4471-b33c-3e01a66388f1" />

### Expected behavior

Images should be rendered if there are attributes/variables in the alt text.

### Steps to reproduce

1. Build the machine-learning.md doc set.
2. Go to http://localhost:3000/ml-dfa-regression.html

### Tooling

- [x] docs-builder
- [ ] migration tooling
- [ ] I'm not sure

**Comment by bmorelli25** [comment]
This is fixed.

<img width="922" alt="Image" src="https://github.com/user-attachments/assets/095e765f-4d14-4c9d-bf53-998c06662ed4" />
